resources/views/admin/index.twig

```diff
@@ -74,5 +73,7 @@
 {% block scripts %}
     {{ parent() }}
     <script src="/assets/js/admin/querystring.js"></script>
-    <script src="/assets/js/admin/talk.js"></script>
+    <script src="/assets/js/talk/talk.js"></script>
+    <script src="/assets/js/talk/listeners.js"></script>
```

Comment:
@BackEndTea @mdwheele 

What do you think about bundling up the scripts via WebPack later on (out of scope of this PR, I believe)?